esp/public/media/scripts/program/modules/scheduling_checks.jsx

```diff
@@ -114,3 +148,71 @@ var RefreshButton = React.createClass({
     </button>;
   },
 });
+
+
+// Modified from react-json-table example code.
+// Required props: rows, saveState (with sort and greyed attributes), settings (with header attribute)
+var SelectTable = React.createClass({
+  getInitialState: function(){
+    // We will store the sorted column and whether each row is greyed out
+    return {sort: this.props.saveState.sort, greyed : this.props.saveState.greyed};
+  },  
+  render: function(){
+    var me = this,
+        // clone the rows
+        items = this.props.rows.slice()
+    ;
+    // Sort the table
+    if( this.state.sort ){
+      items.sort( function( a, b ){
+         return a[ me.state.sort ] > b[ me.state.sort ] ? 1 : -1;
+      });
+    }
+      
+    if (this.props.columns == undefined) {
+      return <JsonTable 
```

Comment:
Rather than writing out the whole thing twice, it [looks like](https://github.com/arqex/react-json-table/blob/master/rjt.js#L93) if you just pass the undefined through, things should work correctly.